### PR TITLE
Bump tektoncd/cli to 78665b7ea758

### PR DESCRIPTION
**Pull request by adambkaplan**
Bumping tektoncd/cli to more recent main commit, to bring in the pod template v1beta1 API fix. This will unblock rebase work, which bumps tekton pipelines to v0.39.0.

### CODE REVIEW

**Comment by adambkaplan** [comment]
cc @avinal - this is what you mentioned during stand-up today. I found that the pod template issue blocked my efforts to rebase this branch.

File: go.mod [hunk 1]
```diff
@@ -6,63 +6,64 @@ require (
...
 	github.com/jonboulle/clockwork v0.3.0
 	github.com/mattn/go-sqlite3 v1.14.5
-	github.com/prometheus/client_golang v1.12.1
-	github.com/spf13/viper v1.12.0
-	github.com/tektoncd/cli v0.25.0
-	github.com/tektoncd/pipeline v0.37.2
+	github.com/prometheus/client_golang v1.13.0
+	github.com/spf13/viper v1.13.0
+	github.com/tektoncd/cli v0.3.1-0.20220928143519-78665b7ea758
+	github.com/tektoncd/pipeline v0.40.0
```
Comment (avinal) [inline go.mod:19]: Do we want to use the latest pipeline version (v0.40.1) or as per cli requirements (v0.40.0)?

File: go.mod [hunk 2]
```diff
@@ -6,63 +6,64 @@ require (
...
+	google.golang.org/protobuf v1.28.1
 	gorm.io/driver/mysql v1.3.3
 	gorm.io/driver/postgres v1.3.4
 	gorm.io/driver/sqlite v1.1.4
 	gorm.io/gorm v1.23.1
 	k8s.io/api v0.23.9
 	k8s.io/apimachinery v0.23.9
 	k8s.io/client-go v1.5.2
-	knative.dev/pkg v0.0.0-20220329144915-0a1ec2e0d46c
+	knative.dev/pkg v0.0.0-20220818004048-4a03844c0b15
```
Comment (avinal) [inline go.mod:34]: This version of knative doesn't work with kcp-changes, but I guess that we can change in the PR bringing kcp-aware changes.

**Review by avinal — APPROVED** [review]
The changes look good to me :)

**Review by avinal — COMMENTED** [review]
It seems the new image includes a `vendor` directory. I was not able to find how/where these images were pushed, probably adding `-mod=readonly` will be good. Here is the log for both api/watcher

```bash
go: inconsistent vendoring in /go/src/github.com/openshift/origin:
	github.com/fatih/color@v1.13.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	github.com/golang/protobuf@v1.5.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
        ...
        ...    
	sigs.k8s.io/structured-merge-diff/v4@v4.2.1: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/client-go: is replaced in go.mod, but not marked as replaced in vendor/modules.txt

	To ignore the vendor directory, use -mod=readonly or -mod=mod.
	To sync the vendor directory, run:
		go mod vendor
subprocess exited with status 1
subprocess exited with status 1
error building at STEP "RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o watcher cmd/watcher/main.go": exit status 1
```

**Comment by sm43** [comment]
sorry for spamming the pr, I am looking into the PAC failure to report pipelinerun status. might trigger the CI a few times.

/retest

**Comment by avinal** [comment]
No Longer required. Closing

**Pull request closed without merging**